flink-connectors/flink-connector-hive/src/test/java/org/apache/flink/table/functions/hive/HiveSimpleUDFTest.java

```diff
@@ -133,7 +134,7 @@ public void testUDFWeekOfYear() throws FlinkHiveUDFException {
 
 		assertEquals(29, udf.eval("1969-07-20"));
 		assertEquals(29, udf.eval(HiveReflectionUtils.convertToHiveDate(hiveShim, "1969-07-20")));
-		assertEquals(29, udf.eval(HiveReflectionUtils.convertToHiveTimestamp(hiveShim, "1969-07-20 00:00:00")));
+		assertEquals(29, udf.eval(HiveReflectionUtils.toHiveTimestamp(hiveShim, Timestamp.valueOf("1969-07-20 00:00:00"))));
```

Comment:
Please add tests to `HiveCatalogUseBlinkITCase` too. Because that is it case. And now there is no it case to cover UDX.